openwisp-qa-check

```diff
@@ -60,6 +67,45 @@ runcheckendline() {
   fi
 }
 
+runstylelintcheck() {
+  package='stylelint'
+  if which node > /dev/null
+    then
+      if [ `npm list -g | grep -c $package` -eq 0 ]; then
```

Comment:
What if I have `npm`? Does `npm` come bundled with `node`?
What do you think about using `whereis stylelint` instead? 
`whereis` comes out of the box and works great in such situations!

Reply:
I will check this and let you know

Reply:
@atb00ker How should I use ``whereis`` for this situation?

Reply:
@niteshsinha17 What do you mean? What's the challenge? Can you please elaborate?

Reply:
Yes `` which stylelint`` will work fine. 
I am getting how to write it with ``whereis`` I tried multiple ways but it was not working as expected. Can you please share exact syntax with ``whereis``

Reply:
Some version of `whereis stylelint | grep /stylelint` should work but don't worry about it, let's focus on `which` only, `whereis` is not a requirement. :smile: